InnoGotchi.API/Tests/Services/AuthenticationServiceTests.cs

```diff
@@ -0,0 +1,142 @@
+﻿using Infrastructure.Services;
+using InnoGotchi.Application.Contracts.Repositories;
+using InnoGotchi.Application.Contracts.Services;
+using InnoGotchi.Application.DataTransferObjects;
+using InnoGotchi.Application.DataTransferObjects.User;
+using InnoGotchi.Application.Exceptions;
+using InnoGotchi.Domain.Entities;
+
+namespace Tests.Services
+{
+    public class AuthenticationServiceTests
+    {
+        private readonly Fixture _fixture;
+        public AuthenticationServiceTests()
+        {
+            _fixture = new Fixture();
+        }
+
+        [Fact]
+        public void VerifyPasswordHash_ReturnsTrue_WhenPasswordAndItsHashAreValid()
+        {
+            //Arrange
+            var passwodHash = "$2b$10$gj7WVHAVH28V5ZNYde0y0.nx7PY7/i1wvNZ91B6bhUnj48FWcEh/a";
```

Comment:
can you extract this to the variable of a class?

Reply:
good idea, made